R/functions-ui.R

```diff
@@ -269,6 +269,114 @@ seed_input <- function(model_name) {
     )
 }
 
+# Helper function to validate a given model file
+#' @rdname epiworldrshiny-ui
+#' @param model_filename The name of the model file to be validated.
+#' @return
```

Comment:
To keep the roxygen tags but avoid creating a manual, this is what you do.

```suggestion
#' @noRd
#' @return
```